<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/iconos/othello13.png</normaloff>:/iconos/othello13.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-image: url(:/iconos/Fondo.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="botonJugar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: transparent;
	color: beige;
    border-style: solid;
    border-width: 2px;
    border-radius: 20px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton:hover:!pressed
{
	border: 3px solid black;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Jugar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonInstrucciones">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: transparent;
	color:beige;
    border-style: solid;
    border-width: 2px;
    border-radius: 20px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton:hover:!pressed
{
	border: 3px solid black;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Instrucciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitulo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; color:#f5f5dc;&quot;&gt;Othello&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImagen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	image: url(:/iconos/Othello.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>81</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser
{
	background-color: rgb(0, 109, 0);
	border-style: solid;
    border-width: 2px;
    border-radius: 12px;
    border-color: beige;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	El othello es un juego para dos personas. Se juega sobre un tablero semejante al de ajedrez, de 8x8, con dos tipos de fichas: negras y blancas. Al jugador uno le corresponden las fichas negras, mientras que al dos, las blancas. El juego comienza siempre con quien posee las fichas negras. Cada usuario, respetando una serie de reglas que veremos a continuación, coloca una ficha de forma alternada (primero el negro, y luego el blanco, sucesivamente). La partida acaba cuando el tablero está completo, o ningún jugador puede mover. El ganador es aquel que más fichas tiene.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	La situación inicial del juego la podemos ver en la siguiente imagen:&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	IMAGENNNNNNNN&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	Como mencionamos, se juega por turnos alternados, y en cada turno el jugador debe poner una ficha en una posición del tablero en la que flanquee o &amp;quot;coma&amp;quot; al menos una ficha del oponente. Veamos cuando un jugador flanquea una ficha, y cuando no.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	Supongamos que ponemos una ficha negra. Luego, desde esa posicion recorremos el tablero en cualquier dirección posible (arriba, abajo, izquierda, derecha, diagonal arrriba izquierda, diagonal arriba derecha, diagonal abajo izquierda y diagonal abajo derecha) hasta encontrar una nueva ficha negra. Si entre las mismas, existen fichas blancas, se las da  vuelta, pasando a ser negras. A esto se le llama flanquear fichas blancas.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	En la proxima imagen, se observa la colocación de una ficha negra de forma incorrecta. El jugador no puede mover en esa posición, porque no flanquea ninguna ficha blanca.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	IMAGENNNNNNN&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;	Sin embargo, el siguiente caso, describe un posible movimiento negro correcto, ya que si recorremos hacia la derecha, vemos que una ficha blanca quedo encerrada entre dos negras.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;IMAGENNNNNNN&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;Por lo tanto, se flanquea la ficha blanca, retornando la siguiente configuración del tablero.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffdd;&quot;&gt;IMAGENNNNN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: transparent;
	color:beige;
    border-style: solid;
    border-width: 2px;
    border-radius: 12px;
    border-color: beige;
    font: bold 12px;
}

QPushButton:hover:!pressed
{
	border: 3px solid black;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Atrás</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
